File: ant-sdk/antd-py/examples/01_connect.py
"""Example 01: Connect to antd daemon and check health.

Prerequisite: antd daemon running locally (default: http://localhost:8082).
"""

from antd import AntdClient

client = AntdClient()
status = client.health()

print(f"Daemon healthy: {status.ok}")
print(f"Network: {status.network}")

if not status.ok:
    print("ERROR: antd daemon is not healthy")
    raise SystemExit(1)

print("Connection OK!")

File: ant-sdk/antd-js/examples/01-connect.ts
/**
 * Example 01: Connect to antd daemon and check health.
 *
 * Prerequisite: antd daemon running locally (default: http://localhost:8082).
 */

import { createClient } from "../src/index.js";

const client = createClient();
const status = await client.health();

console.log(`Daemon healthy: ${status.ok}`);
console.log(`Network: ${status.network}`);

if (!status.ok) {
  console.log("ERROR: antd daemon is not healthy");
  process.exit(1);
}

console.log("Connection OK!");
